polytope/polytope.py

```diff
@@ -570,6 +564,18 @@ def _rotate(polyreg, i=None, j=None, u=None, v=None, theta=None, R=None):
     return R
 
 
+def _Givens_rotation_matrix(i, j, theta, N):
```

Comment:
Moving some of the functions to a different file would save me from having to scroll so much!